src/components/ui/button.tsx

```diff
@@ -146,6 +146,10 @@ const page = StyleSheet.create({
   secondDisabledContainer: {
     backgroundColor: GRAPHIC_SECOND_1,
   },
+  text: {
```

Comment:
for that these fix?

Reply:
yes, but these problem were solved by another commit( sorry(

Reply:
As practice shows, it is better to edit such things very carefully and when there is normal access to the design